nextcloudappstore/scaffolding/forms.py

```diff
@@ -58,8 +59,6 @@ class AppScaffoldingForm(Form):
 class IntegrationScaffoldingForm(Form):
     name = CharField(max_length=80, label=_('Integration name'),
                      widget=TextInput())
-    author_name = CharField(max_length=80, label=_('Author\'s full name'))
-    author_email = EmailField(label=_('Author\'s e-mail'))
```

Comment:
remove intentionally?

Reply:
Yes. It wasn't used - the full name and email are re-used from the account that submits it.